src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -142,6 +143,7 @@
 
             <tag name="kernel.event_listener" event="kernel.request" method="onKernelRequest" priority="2" />
         </service>
+        <service id="ApiPlatform\Core\EventListener\DeserializeListener" alias="api_platform.listener.request.deserialize"/>
```

Comment:
When can it be necessary to inject this class?

Reply:
It does trigger a notice, So we should look